byterun/backtrace.c

```diff
@@ -152,6 +154,35 @@ CAMLprim value caml_get_exception_raw_backtrace(value unit)
   CAMLreturn(res);
 }
 
+/* Copy back a backtrace and exception to the global state */
+/* noalloc (caml value): so no CAMLparam* CAMLreturn* */
```

Comment:
@bobot : I was thinking of commenting *here*, to avoid people calling this function on arrays they have created themselves on the OCaml side.